sns/main.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_sns_topic" "log_aggregation" {
```

Comment:
For the sns topic module, is the idea to add additional sns topics to this main.tf?

Reply:
The idea is to separately call the SNS module for each new SNS topic, with appropriate variables - the main.tf in the modules only needs to contain a single SNS topic.

Reply:
OK. So should the resource be name something more generic, rather than "log_aggregation"?

Reply:
agreed, now implemented